tests/lost_reset/test

```diff
@@ -78,6 +90,12 @@ while ($line = <$fh_out>) {
 ok($found_msg, 1); # Was the message well-formed?
 ok($result_lost == $lost); # Do the two lost values agree?
 
+if (!$result_lost || !$lost || $result_lost != $lost) {
```

Comment:
You know my feelings on stuff like this (I don't like it), but let's solve this generically instead of adhoc and per-test, see the issue https://github.com/linux-audit/audit-testsuite/issues/54.